Woof/Woof/Model/Sitter.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+/**
+ A person who can walk with pet (or several pets) and provide service of walking for pet owners.
+ Conforms to the `Codable` and `Identifiable` protocols.
+ */
+struct Sitter: Codable, Identifiable {
+    /// The unique identifier for the sitter.
+    let id: UUID
+
+    /// The name of the sitter.
+    var name: String
+
+    /// The surname of the sitter.
+    var surname: String
+
+    /// The phone number of the sitter.
+    var phone: String
+
+    /// The URL of the sitter's avatar image.
+    var avatarUrl: URL?
+
+    /// The biography or description of the sitter.
```

Comment:
nit:
```suggestion
    /// Additional Information about sitter.
    /// The dog walker can describe their experience with dogs, favorite places to run, and any other important details
    /// that will help them better understand how to meet the needs and desires of each owner's dog
    /// They can also add information about their special skills and experience, which can be useful during the outings.
```
I'm not sure about such a long comment.

Reply:
@iBYarchitect How about this?   
/// The additional information about the sitter, like his experience with dogs, favorite places for walks, special skills, certificates, etc.